packages/sdk/src/react/hooks.ts

```
@@ -89,205 +95,178 @@ export function useEntityId(...keys: BigNumberish[]): BigNumberish {
-                    isUpdating.current = false;
-                });
-        }
+/**
+ * Subscribe to event changes. This hook fetches initial events from torii and subscribes to new events.
+ *
+ * @param query ToriiQuery
+ */
+export function useEventQuery<Schema extends SchemaType>(
+    query: ToriiQueryBuilder<Schema>
+) {
+    const { sdk, useDojoStore } = useDojoSDK<() => any, Schema>();
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)